repos/system_upgrade/el7toel8/libraries/tests/test_isccfg.py

```diff
@@ -343,5 +345,22 @@ def test_walk():
     assert 'dnssec-validation' not in state
 
 
+def test_empty_config():
+    """ Test empty configuration """
+
+    callbacks = {
```

Comment:
for empty file no callbacks make sense or are needed. Use just ``callbacks = {}``.